.github/CODEOWNERS

```diff
@@ -71,22 +105,52 @@
 /docs/framework/configure-apps/file-schema/wcf/** @HongGit
 /docs/framework/data/wcf/** @HongGit
 /docs/framework/wcf/** @HongGit
-# WPF
-/docs/framework/wpf/** @Thraka
-# Winforms
-/docs/framework/winforms/** @Thraka
 
 ################## .NET STANDARD ##################
-# Analyzers:
-/docs/standard/analyzers/** @BillWagner
+# Analyzers
+/docs/standard/analyzers/** @mairaw
+# Assembly
+/docs/standard/assembly/** @gewarren
+# Asynchronous Programming Patterns
+/docs/standard/asynchronous-programming-patterns/** @gewarren
+# Attributes
+/docs/standard/attributes/** @tdykstra
+# Base types
+/docs/standard/base-types/** @Thraka
+# Collections
+/docs/standard/base-types/** @BillWagner
+# Base types
+/docs/standard/base-types/** @Thraka
+# Data XML
+/docs/standard/data-xml/** @gewarren
+# Datetime
+/docs/standard/datetime/** @Thraka
+# Events
```

Comment:
design-guidelines folder seems to be missing here.

Reply:
I skipped the self-serve folders on this round. Let me add them.

Reply:
I put my name for that one. We don't accept PRs for that content (unless you're fixing some kind of formatting or something) since it's copyrighted content.